### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'getElementsByTagName'

### ISSUE THREAD

**Issue by amangupta87** [tdoly/apk_parse#4]
Though test.py runs fine for majority of apps but for some apps, I am encountering following error:
`  File "/home/aman/apk_parse-master/apk.py", line 471, in get_services
    return self.get_elements("service", "name")
  File "/home/aman/apk_parse-master/apk.py", line 393, in get_elements
    for item in self.xml[i].getElementsByTagName(tag_name):
AttributeError: 'NoneType' object has no attribute 'getElementsByTagName'`

So basically some issue is there in apk.py.
Please help me to solve this issue.
Thanks in advance.
